docs/requirement-sets/common/shared-runtime-requirement-sets.md

```diff
@@ -12,10 +12,11 @@ Requirement sets are named groups of API members. Office Add-ins use requirement
 
 Parts of an Office Add-in that run JavaScript code, such as task panes, function files launched from add-in commands, and Excel custom functions, can share a single JavaScript runtime. This enables all the parts to share a set of global variables, to share a set of loaded libraries, and to communicate with each other without having to pass messages through a persisted storage. For more information, see [Configure your Office Add-in to use a shared JavaScript runtime](/office/dev/add-ins/develop/configure-your-add-in-to-use-a-shared-runtime).
 
-The following table lists the SharedRuntime 1.1 requirement set, the supported Office client applications, and the **minimum** builds or versions for those applications where applicable.
+The following table lists the SharedRuntime 1.1 and 1.2 requirement sets, the supported Office client applications, and the **minimum** builds or versions for those applications where applicable.
 
 | Requirement set | Office on Windows<br>- Microsoft 365 subscription<br>- retail perpetual Office 2016 and later | Office on Windows<br>(volume-licensed perpetual) | Office on Mac | Office on iPad | Office on the web | Office Online Server |
 |:-----|:-----|:-----|:-----|:-----|:-----|:-----|
+| SharedRuntime 1.2 | Excel: Version 2108 (Build 14326.20508) | Not supported | Excel 16.52.0.0 | Not supported | Excel: Supported | Not supported |
 | SharedRuntime 1.1  | Excel: Version 2002 (Build 12527.20092)<br><br>PowerPoint: Version 2102 (Build 13722.10000)<br><br>Word: Version 2205 (Build 15202.10000) | Excel 2021: Version 2108 (Build 12527.20092)<br><br>PowerPoint 2021: Version 2108 (13722.10000) | Excel: 16.35<br><br>PowerPoint: 16.46.120.0<br><br>Word: 16.61.401.0 | Not supported | Excel, PowerPoint, Word: Supported | Not supported |
 
 ## Office versions and build numbers
```

Comment:
Instead of listing what's in the reqt. sets on the common requirement sets page, consider listing them on this page similar to [Ribbon API](https://learn.microsoft.com/en-us/javascript/api/requirement-sets/common/ribbon-api-requirement-sets?view=common-js#ribbon-api-11). So you'd add SharedRuntime 1.1 and SharedRuntime 1.2 sections.

Reply:
Thanks for the review! I added the sections like you recommended. Ready for you to take another look anytime.